components/oracle.tsx

```diff
@@ -105,7 +107,7 @@ export default function Oracle({ children }) {
       <div className="container p-footer">
         <div className="row justify-content-center">
           <div className="col-md-10 mt-2">
-            <CardBecomeCouncil />
+            {!wallet?.isCouncil && <CardBecomeCouncil />}
```

Comment:
Won't this also show if you don't connect a wallet ?

Reply:
not. It displays even without linked wallet